index.js

```diff
@@ -1,29 +1,25 @@
 'use strict';
-var net = require('net');
-var onetime = require('onetime');
+const net = require('net');
 
-module.exports = function (port, opts, cb) {
-	if (typeof opts !== 'object') {
-		cb = opts;
-		opts = {};
-	}
+module.exports = (port, opts) => {
+	opts = Object.assign({}, opts);
```

Comment:
Don't see a reason why to use `Object.assign` here because we only read data from the object. Although might be a best practice to always do this. Like to know your guys opinion on that.